openedx/core/lib/celery/routers.py

```diff
@@ -49,7 +49,7 @@ def ensure_queue_env(self, desired_env):
             (
                 queue
                 for queue in queues
-                if '.{}.'.format(desired_env) in queue
+                if '.{}'.format(desired_env) in queue
```

Comment:
This allows us to specify an entire routing key suffix in the `alternate_env_tasks()` router method.  Without doing that, the `ensure_queue_env()` method will just give us the first queue in which ".lms." appears, which was fine when there was only one alternate queue available, but now there are two.